src/com/google/javascript/jscomp/SourceFileMapping.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2015 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.javascript.jscomp;
+
+import com.google.debugging.sourcemap.proto.Mapping.OriginalMapping;
+import javax.annotation.Nullable;
+
+/**
+ * A SourceFileMapping maps a source file, line, and column into an {@link OriginalMapping}.
+ *
+ * @see com.google.debugging.sourcemap.SourceMapping
+ */
+public interface SourceFileMapping {
+  /**
+   * Returns the original mapping for the file name, line number and column position found
+   * in the source map. Returns {@code null} if none is found.
+   *
+   * @param lineNumber The line number, 1-based.
```

Comment:
need to change this to `lineNo` and the next line to `columnNo` so that they match the param names on line 35. I think after that the `javadoc` command will pass.